lib/gc/ExecutionEngine/CPURuntime/Microkernel/BrgemmOnednn.cpp

```diff
@@ -77,35 +77,36 @@ int64_t dnnl_brgemm_dispatch(int64_t M, int64_t N, int64_t K, int64_t LDA,
                              int64_t LDB, int64_t LDC, int64_t stride_a,
                              int64_t stride_b, float beta, int64_t dtypeA,
                              int64_t dtypeB) {
-  brgemm_desc_t desc;
+  std::shared_ptr<brgemm_desc_t> desc_ptr = std::make_shared<brgemm_desc_t>();
```

Comment:
@huanghaixin008 how about in this way, store the desc_ptr in a shared_ptr

Reply:
It's ok, but we need to change ptr of desc in `brgemm_cache_info_t` to shared_ptr as well, or the ptr would be released after the dispatch func.